Rollbar/RollbarNotifier.m

```diff
@@ -385,6 +399,7 @@ - (void)queuePayload:(NSDictionary*)payload {
     [fileHandle writeData:[NSJSONSerialization dataWithJSONObject:payload options:0 error:nil safe:true]];
     [fileHandle writeData:[@"\n" dataUsingEncoding:NSUTF8StringEncoding]];
     [fileHandle closeFile];
+    [[RollbarTelemetry sharedInstance] clearAllData];
```

Comment:
Why are you doing this?

Reply:
My understanding of this is the payload (including telemetry data) has been written to the disk. So this clears the telemetry data for the next notification. Now that I think about it, it might make more sense to keep them and send it along with the next notification as well. Is that the preferred way to handle it?